packages/app/src/cli/models/app/loader.test.ts

```diff
@@ -578,7 +578,19 @@ wrong = "property"
     await writeWebConfiguration({webDirectory: anotherWebDirectory, role: 'backend'})
 
     // Then
-    await expect(loadTestingApp()).rejects.toThrow()
+    try {
+      await loadTestingApp()
+      expect.fail('Expected loadTestingApp to throw an error')
+    } catch (error) {
+      if (!(error instanceof AbortError)) {
+        throw error
+      }
+      expect(error.message).toContain('You can only have one "web" configuration file')
+      expect(error.message).toContain('backend')
```

Comment:
Reading just this test case might lead someone to believe that `You can only have one "web" configuration file. (full stop)`

I'd merge these two lines. Same below.